### PR TITLE
Add Gazebo site config package

### PR DESCRIPTION
**Pull request by sea-bass**
This PR adds a Gazebo based site configuration for an upcoming tutorial!

Also had some licensing touch-ups to ensure we're uniformly on BSD-3.

To test, follow what you normally would do, but use the `picknik_ur_gazebo_config` package.

### CODE REVIEW

File: src/picknik_ur_gazebo_config/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# picknik_ur_gazebo_config
+
+A MoveIt Studio site configuration for UR5e simulated in Gazebo, based on the `picknik_ur_base_config` package available [here](https://github.com/PickNikRobotics/ur5e_sim_base_config).
```
Comment (cmelliott2) [inline src/picknik_ur_gazebo_config/README.md:3]:
```suggestion
A MoveIt Studio site configuration for a UR5e simulated in Gazebo, based on the `picknik_ur_base_config` package available [here](https://github.com/PickNikRobotics/ur5e_sim_base_config).
```

File: src/picknik_ur_site_config/README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # picknik_ur_site_config
 
-A MoveIt Studio site configuration for UR5e based on the picknik_ur_base_config package available [here](https://github.com/PickNikRobotics/ur5e_sim_base_config). This package follows the required structure described in [Creating a New Site Config Package](https://docs.picknik.ai/en/stable/concepts/config_package/config_package.html#creating-a-new-site-config-package), and may be used to add environments, behaviors, and objectives on top of the UR5e base configuration specified in `site_config.yaml`
+A MoveIt Studio site configuration for UR5e based on the `picknik_ur_base_config` package available [here](https://github.com/PickNikRobotics/ur5e_sim_base_config).
```
Comment (cmelliott2) [inline src/picknik_ur_site_config/README.md:3]:
```suggestion
A MoveIt Studio site configuration for a UR5e based on the `picknik_ur_base_config` package available [here](https://github.com/PickNikRobotics/ur5e_sim_base_config).
```

File: src/picknik_ur_site_config/README.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
...
+
+This package follows the required structure described in [Creating a New Site Config Package](https://docs.picknik.ai/en/stable/concepts/config_package/config_package.html#creating-a-new-site-config-package), and may be used to add environments, behaviors, and objectives on top of the UR5e base configuration specified in `site_config.yaml`
```
Comment (cmelliott2) [inline src/picknik_ur_site_config/README.md:5]: I think everywhere else in the tutorials Objectives and Behaviors are capitalized

File: src/picknik_ur_gazebo_config/README.md (hunk 1 above)
Comment (sea-bass) [reply src/picknik_ur_gazebo_config/README.md:3]: The URL is actually wrong too now that you highlighted this. Will fix.

**Review by Abishalini — APPROVED** [review]
Tested all the objectives and endpoint/joint control. 
Looks good to me

**Comment by sea-bass** [comment]
just gonna YOLO merge this since I won't have time to keep working on this config.

**Pull request merged**